### DISCUSSION TITLE
LGT8F328P-LQFP32  purple Board

### DISCUSSION THREAD

**Discussion by herbkonik** [dbuezas/lgt8fx#222] [General]
Just to let you know... :

This days i got a few of this purple LGT8F328P-LQFP32 Boards with the USB C conector.
![LGT8F328P-LQFP32](https://user-images.githubusercontent.com/112805193/191808701-e69963b6-669b-4b67-9eeb-8ba003ce4b58.png)

Like expected it works like the green boards, - with a pleasant surprise:
This Board has a significantly lower power consumption compared with the green Nano style board.  While my Greens too take about 32mA @32Mhz and 28mA @16Mhz (like the Wiki says) the purple takes only about 16mA @32Mhz, 11mA @16Mhz !

**Comment by dbuezas** [comment]
Oh, wow! Almost like the mini-style variant!

**Comment by trimarco232** [comment]
[https://github.com/dbuezas/lgt8fx/discussions/187](url)

**Reply by herbkonik** [reply]
at your link i get a page not existing error...

Edit: OK, found the thread
